### DISCUSSION TITLE
distribute coolify to docker-hub

### DISCUSSION THREAD

**Discussion by SlickMoose** [coollabsio/coolify#3632] [General]
Hey, been trying to self host coolify inside my workplace, but my employeer for some reason blocked ghcr.io repo but not the docker hub. Coolify image was last updated on docker hub with version 3.12, but coolify-realtime is available only on ghcr. 

Is it possible to upload most stable recent version of both coolify and coolify-realtime to docker hub?

Thanks in advance

**Comment by peaklabs-dev** [comment]
This will be possible soon as we will be uploading all images to docker hub as well as a mirror and to get IPv6 support.

**Reply by SlickMoose** [reply]
For POC will it be ok if I create fork repo on docker hub?

**Reply by peaklabs-dev** [reply]
Sure, feel free to do so, you could also PR with the updated GH actions for image building which would help us integrate it faster.
